src/lib/Behat/PageElement/Element.php

```diff
@@ -14,14 +14,17 @@
     /** @var int */
     public $defaultTimeout = 5;
 
-    /* \EzSystems\EzPlatformAdminUi\Behat\UtilityContext */
+    /* \EzSystems\EzPlatformAdminUi\Behat\Helper\UtilityContext */
     protected $context;
+
     protected $fields;
 
     public function __construct(UtilityContext $context)
     {
         $this->context = $context;
     }
 
-    abstract public function verifyVisibility(): void;
+    public function verifyVisibility(): void
```

Comment:
I've reverted that change, it would require too much changes in other repositories at this moment.